apps/modernization-ui/src/components/AutoCompleteInput/AutoCompleteInput.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComboBox, Label } from '@trussworks/react-uswds';
+import { InputMaybe } from 'generated/graphql/schema';
+
+type AutoCompleteProps = {
+    label: string;
+    name: string;
+    value: InputMaybe<string> | undefined;
+    onChange: (event: any) => void;
+    options: any;
```

Comment:
`SelectInput`  defines options as `{ name: string; value: string }`, let's try to keep them the same